### PR TITLE
Reshaped config object

### PR DESCRIPTION
**Pull request by harshkothari410**

### CODE REVIEW

File: src/commands/start.js [hunk 1]
```diff
@@ -28,24 +28,24 @@ function execute(collectorName, refocusUrl, accessToken) {
...
     /*
      * TODO: Replace the success/failure/progress listeners here with proper
      * logging once we have heartbeat
      */
     repeater.create({
       name: 'Heartbeat',
-      interval: config.collectorConfig.heartbeatInterval,
-      func: sendHeartbeat,
+      interval: config.refocus.heartbeatInterval,
+      func: () => sendHeartbeat(config.collector),
```
Comment (iamigo) [inline src/commands/start.js]: I see you're sending the arg here but the sendHeartbeat function definition in heartbeat.js isn't actually using it.

**Comment by pallavi2209** [comment]
The changes look good. @harshkothari410 @iamigo Do you think it makes sense to document the updated config shape either in comments or Readme. It would really help team members to refer back to the structure easily whenever needed.

**Comment by harshkothari410** [comment]
@pallavi2209 I guess its good idea. But where should we document this?

**Comment by pallavi2209** [comment]
I would say Readme would be best unless there is reason not to document there. Thoughts?

**Comment by iamigo** [comment]
since someone just installing or using a collector doesn't *need* to know the internal implementation details of how we store this info in memory, maybe just describe it in comment in config/config.js rather than the README

**Comment by harshkothari410** [comment]
README is not for writing down our internal code flow I guess. Its open repo, so it should contain how to use this thing. But yeah we can make configObject.md file for this.

**Comment by pallavi2209** [comment]
Sounds good. configObject.md or comment, either way is fine.

**Comment by harshkothari410** [comment]
@pallavi2209 @iamigo updated as comment. Please review now

**Pull request merged**